content/practices/landmark-regions/landmark-regions-practice.html

```diff
@@ -80,91 +80,91 @@ <h2>HTML Sectioning Elements</h2>
+          <li><code>banner</code>、<code>main</code>、<code>complementary</code>、<code>contentinfo</code>のランドマークは、トップレベルのランドマークであるべきです。</li>
 
-          <li>Landmark roles can be nested to identify parent/child relationships of the information being presented.</li>
+          <li>ランドマークロールは、提示される情報の親子関係を識別するためにネストすることができます。</li>
         </ul>
 
-        <p id="aria_lh_step3"><strong>Step 3: Label areas</strong></p>
+        <p id="aria_lh_step3"><strong>ステップ3: エリアにラベルを付ける</strong></p>
 
         <ul>
           <li>
-            If a specific landmark role is used more than once on a page, provide each instance of that landmark with a unique label.
-            There is one rare circumstance where providing the same label to multiple instances of a landmark can be beneficial: the content and purpose of each instance is identical.
-            For example, a large search results table has two sets of identical pagination controls -- one above and one below the table, so each set is in a navigation region labelled <q>Search Results</q>.
-            In this case, adding extra information to the label that distinguishes the two instances may be more distracting than helpful.
+            特定のランドマークロールがページ上で複数回使用される場合、そのランドマークの各インスタンスに固有のラベルを提供します。
+            複数のインスタンスに同じラベルを提供することが有益な稀な状況が一つあります: 各インスタンスのコンテンツと目的が同一である場合。
+            例: 大きな検索結果テーブルが、テーブルの上下に同一のページネーションコントロールを2セット持っているため、各セットは<q>検索結果</q>というラベルのナビゲーション領域にあります。
+            この場合、2つのインスタンスを区別するためにラベルに追加情報を追加することは、役立つよりもかえって混乱を招く可能性があります。
           </li>
 
           <li>
-            If a landmark is only used once on the page it may not require a label.
-            See Landmark Roles section below.
+            ランドマークがページ上で一度しか使用されない場合、ラベルを必要としない場合があります。
+            以下のランドマークロールセクションを参照してください。
           </li>
 
-          <li>If an area begins with a heading element (e.g. <code>h1-h6</code>) it can be used as the label for the area using the <code>aria-labelledby</code> attribute.</li>
+          <li>エリアが見出し要素（例：<code>h1-h6</code>）で始まる場合、<code>aria-labelledby</code>属性を使用してエリアのラベルとして使用することができます。</li>
 
-          <li>If an area requires a label and does not have a heading element, provide a label using the <code>aria-label</code> attribute.</li>
+          <li>エリアがラベルを必要とし、見出し要素を持っていない場合、<code>aria-label</code>属性を使用してラベルを提供します。</li>
 
           <li>
-            Do not use the landmark role as part of the label.
-            For example, a navigation landmark with a label &quot;Site Navigation&quot; will be announced by a screen reader as &quot;Site Navigation Navigation&quot;.
-            The label should simply be &quot;Site&quot;.
+            ラベルの一部としてランドマークロールを使用しないでください。
+            例えば、ラベルが「サイトナビゲーション」のナビゲーションランドマークは、スクリーンリーダーによって「サイトナビゲーションナビゲーション」として発表されます。
+            ラベルは単に「サイト」であるべきです。
           </li>
         </ul>
       </section>
 
       <section>
-        <h2>Landmark Roles</h2>
+        <h2>ランドマークの役割</h2>
```

Comment:
ここからは各ロールの説明かと思いますので
```suggestion
        <h2>ランドマークロール</h2>
```